### DISCUSSION TITLE
[NMS][helm] Why are we using latest tag on nginx image for NMS Proxy?

### DISCUSSION THREAD

**Discussion by ShubhamTatvamasi** [magma/magma#5197] [General]
Why are we using `latest` tag on `nginx` image for `nms-nginx-proxy` deployment, can't this lead to some incompatibility issues?
https://github.com/magma/magma/blob/v1.4/orc8r/cloud/helm/orc8r/charts/nms/values.yaml#L89

**Comment by karthiksubraveti** [comment]
I don't know about the history here. My guess is since the actual configs we use is pretty minimal here, it might not affect us much. 
@tmdzk I remember some conversation around pinning version for a release, was that for the gateway alone  ?

**Reply by ShubhamTatvamasi** [reply]
How about we pin `nginx:1.18.0` image, which is the latest stable image from nginx?

**Reply by sudhikan** [reply]
Interesting thread; during troubleshooting your on-prem db issues, I noticed that your setup pulled 1.17.0 while my setup was running 1.16.0 ... I almost figured that could be golden ticket but the change log didn't yield any useful leads... but +1 to pinning and moving up versions as needed.
